.buildkite/pipeline.yml

```diff
@@ -5,7 +5,8 @@ env:
 
 steps:
   - commands:
-    - "nix-shell --run \"cabal v2-update && make\""
+    - "nix build"
+    - "cp -r result/out out"
```

Comment:
Since `out` is coming from the `/nix/store` it's read-only and when the CI agent attempts to clean it for the subsequent CI run, it gets "permission denied"

Reply:
What's the correct way to do this? I just want to preserve the old behavior on buildkite and deployment. The artifacts used to end up in `out`, now they are in `result/out`